### PR TITLE
Modify XGBoostJob example for the new Controller

### PR DESCRIPTION
**Pull request by andreyvelich**
Since we will release Kubeflow Training operator within Kubeflow 1.4, let's make appropriate change for XGBoost example in Katib.

/cc @kubeflow/wg-training-leads

### CODE REVIEW

File: examples/v1beta1/xgboost-lightgbm.yaml [hunk 1]
```diff
@@ -59,11 +54,11 @@ spec:
             template:
               spec:
                 containers:
-                  - name: xgboostjob
+                  - name: xgboost
                     image: docker.io/kubeflowkatib/xgboost-lightgbm:1.0
                     ports:
                       - containerPort: 9991
-                        name: xgboostjob-port
+                        name: xgboost-port
```
Comment (Jeffwan) [inline examples/v1beta1/xgboost-lightgbm.yaml:61]: https://github.com/kubeflow/tf-operator/blob/08c0d0374cada2c18d19ce9fc07bb598e62b6ccf/pkg/apis/xgboost/v1/constants.go#L22

default container port name is still `xgboostjob-port`?

**Review by terrytangyuan — COMMENTED** [review]
LGTM besides @Jeffwan's comment above

File: examples/v1beta1/xgboost-lightgbm.yaml (hunk 1 above)
Comment (andreyvelich) [reply examples/v1beta1/xgboost-lightgbm.yaml]: You are right, thanks!

**Review by gaocegege — COMMENTED** [review]
Thanks for your contribution!

LGTM

**Review by Jeffwan — APPROVED** [review]

**Pull request merged**